apps/api/src/services/archive/index.ts

```
@@ -41,12 +55,194 @@ export class Archive extends Effect.Service<Archive>()(
+                db
+                  .insert(playerWeaponHistoryTable)
+                  .values(playerWeaponsHistory),
+              ],
+              { concurrency: "unbounded" },
+            ).pipe(
+              Effect.tapError((error) => {
+                return Effect.logError(
+                  "Error creating player legend or weapon history",
+                  error,
+                )
+                return Effect.succeed(null)
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)